src/components/panel/panel.js

```diff
@@ -306,6 +309,8 @@ angular
  * Adds a class to the panel. DO NOT use this to hide/show the panel.
  *
  * @param {string} newClass Class to be added.
+ * @param {boolean} toElement Whether or not to add the class to the panel
```

Comment:
Why is this documented at the top of the file and down below on the actual method? Seems to be duplicated?

@ErinCoughlan Is this an internal Google thing to do it twice?